src/ansys_sphinx_theme/search/fuse_search.py

```diff
@@ -100,9 +100,6 @@ def create_search_index(app, exception):
     if exception:
         return
 
-    if not app.config.html_theme_options.get("static_search", {}):
```

Comment:
Just curious about why these lines got removed.

Reply:
These lines were forcing developers to add "static_search" as a parameter in the HTML theme options to create the search.json file. Since it's the default, this isn't necessary, so I removed the line.